modules/user_accounts/php/NDB_Form_user_accounts.class.inc

```diff
@@ -323,6 +341,58 @@ class NDB_Form_User_Accounts extends NDB_Form
         unset($values['examiner_radiologist']);
         //END EXAMINER UPDATE
 
+        // START NOTIFICATIONS UPDATE
+        // get current notifications for user
+        $curr_sub = NDB_Notifier::getUserNotificationModuleServices(
+            $user->getData("ID")
+        );
+
+        if (!$this->isCreatingNewUser()) {
+            // get notification details
+            foreach ($curr_sub as $module => $operations) {
+                foreach ($operations as $operation => $services) {
+                    $module_id = NDB_Notifier::getNotificationModuleID(
+                        $module,
+                        $operation
+                    );
+                    unset($services['desc']);
+                    foreach ($services as $service => $subscribed) {
+                        $service_id = NDB_Notifier::getNotificationServiceID(
+                            $service
+                        );
+                        $var_name   = "notif_" . $module . "_"
+                            . $operation . "_" . $service;
+
+                        if ($values[$var_name] === 'on') {
```

Comment:
@ridz1208 Ok, never-mind. It was only to make 2 transactions instead of N (where n  is #module * #service). But there will never be 1000+ of those.  So it is insignificant.